frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -72,7 +69,18 @@ export const XYChart = ({
     yDomains.left,
     yDomains.right,
   );
+
+  const [xTickValues, setXTickValues] = useState<
+    (string | number | Date)[] | undefined
```

Comment:
nit: such long inline types better to define separately for better readability

Reply:
Agreed.